pkg/common/config/config.go

```diff
@@ -735,3 +735,31 @@ func GetConfigPath(ctx context.Context) string {
 	}
 	return cfgPath
 }
+
+// GetSessionUserAgent returns clusterwise unique useragent
+func GetSessionUserAgent(ctx context.Context, useragent string) string {
```

Comment:
I see all callers are calling  `GetSessionUserAgent` with empty string useragent, then why do we supply useragent to this method?

Reply:
fixed in latest commit